index.html

```diff
@@ -140,50 +140,91 @@ <h2>Why Git?</h2>
-                  <pre><code class="hljs" data-trim>git clone</code></pre>
-                </span>
+                <span>Configuring git
+                  <pre><code class="hljs" data-trim>git config --global user.name "John Doe"</code></pre>
+                  <pre><code class="hljs" data-trim> git config --global user.email "johndoe@example.com"</code></pre>
+                  <pre><code class="hljs" data-trim> git config --global core.editor "nano"</code></pre>
+                  <p>
+                  <ul>
+                      <li class="fragment"> This is done for git to identify who is making the commits, for people who will check later on. We recommend to go with nano in the beginning for the editor of choice.</li>
+                  </ul>
+                  </p>
+                  </span>
                 </section>
                 <section>
-                    <span>Make Changes</span>
+                <span>Obtain Repository <br>[Make your own/Clone from upstream]
+                  <pre><code class="hljs" data-trim>git init</code></pre>
+                  <pre><code class="hljs" data-trim>git clone {repo_url} {directory_name} </code></pre>
+                </span>
                 </section>
                 <section>
-                <span>Stage Changes
+                <span>Stage Changes<br>[Untracked Changes --> Staging them]
                   <pre><code class="hljs" data-trim>git add {filename}</code></pre>
+                  <pre><code class="hljs" data-trim>git status</code></pre>
+                  <p>
+                  <ul>
+                      <li class="fragment">Git tracks everything in your directory. Go ahead make and edit some new files and let git do all the heavy lifting for you.</li>
+                  </ul>
+                  </p>
                 </span>
                 </section>
                 <section>
                 <span>
-                  Commit Changes
+                  Commit Changes <br> [Staged changes --> Commiting them]
                   <pre><code class="hljs" data-trim>git commit -m “Commit message”</code></pre>
+                  <pre><code class="hljs" data-trim>git status</code></pre>
+                  <pre><code class="hljs" data-trim>git log</code></pre>
                 </span>
                 </section>
                 <section>
                 <span>
-                  Push Changes to Remote Repo
-                  <pre><code class="hljs" data-trim>git push {remote_name} {repo_name}</code></pre>
+                  Defining Remotes
+                  <pre><code class="hljs" data-trim>git remote -v</code></pre>
+                  <pre><code class="hljs" data-trim>git remote add {remote_name} {repo_url}</code></pre>
+                  <p>
+                  <ul>
+                      <li class="fragment">Remote is just a fancy way to say where the code is stored on GitHub, and where do you want to push/pull the changes from.</li>
```

Comment:
This is incorrect. Can you explain it in better terms?

Reply:
I think I should... done.